File: 백준/Bronze/2702. 초6 수학/초6 수학.py
import sys
input=lambda:sys.stdin.readline().rstrip()
def gcd(x,y):
    return y if x%y==0 else gcd(y,x%y)
for _ in range(int(input())):
    a,b=map(int,input().split())
    n=gcd(a,b)
    print(a*b//n,n)

File: 백준/Bronze/2702. 초6 수학/초6 수학.c
#include <stdio.h>
int gcd(int x, int y) {
    return x % y == 0 ? y : gcd(y, x % y);
}
int main(void) {
    int t, a, b, n;
    scanf("%d", &t);
    for (int i = 0; i < t; i++) {
        scanf("%d %d", &a, &b);
        n = gcd(a, b);
        printf("%d %d\n", a * b / n, n);
    }
    return 0;
}